adafruit_hts221.py

```diff
@@ -65,21 +71,22 @@ class CV:
     """struct helper"""
 
     @classmethod
-    def add_values(cls, value_tuples):
+    def add_values(
+        cls, value_tuples: Sequence[Tuple[str, int, Union[int, str]]]
```

Comment:
Did you mean this ? `RATE_12_5_HZ` is a float in the code.
```suggestion
        cls, value_tuples: Sequence[Tuple[str, int, Union[int, float]]]
```

Reply:
Yes I did, thank you!